src/main/java/com/team/piickle/ballot/dto/BallotTopicResponseDto.java

```diff
@@ -0,0 +1,19 @@
+package com.team.piickle.ballot.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.team.piickle.ballot.domain.BallotTopic;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class BallotTopicResponseDto {
+    @JsonProperty("_id")
+    private String ballotTopicId;
+    private String title;
+
+    public BallotTopicResponseDto(BallotTopic ballotTopic) {
```

Comment:
생성자 말고 정적팩토리메서드 쓰는 건 어땡?

```suggestion
    public static BallotTopicResponseDto from(BallotTopic ballotTopic) {
```

Reply:
좋습니다!